heartbeat/monitors/plugin_test.go

```diff
@@ -1,210 +0,0 @@
-// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This file was accidentally committed in a past PR and was giving compiler trouble here.